temporalio/worker/_worker.py

```diff
@@ -106,9 +107,11 @@ def __init__(
         *,
         task_queue: str,
         activities: Sequence[Callable] = [],
+        nexus_service_handlers: Sequence[Any] = [],
         workflows: Sequence[Type] = [],
         activity_executor: Optional[concurrent.futures.Executor] = None,
         workflow_task_executor: Optional[concurrent.futures.ThreadPoolExecutor] = None,
+        nexus_task_executor: Optional[concurrent.futures.Executor] = None,
```

Comment:
Is there a `max_concurrent_nexus_tasks` setting we need to add?

Reply:
There is but I believe it's only used by the `WorkerTuner` and we don't have that implemented for Nexus yet. Is that a problem?

https://github.com/temporalio/sdk-python/blob/nexus/temporalio/worker/_worker.py#L474-L479

Reply:
Yes, I think we need to update the worker tuner to support Nexus (it's already in Core)